configure.in: do not hard-code search patch for SSL headers

Do. Not. Do. That. It breaks cross-compilation.

Also use the SSL headers dir to look for blowfish headers.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN vtun-3.0.3.orig/configure.in vtun-3.0.3/configure.in
--- vtun-3.0.3.orig/configure.in	2009-03-29 12:08:51.000000000 +0200
+++ vtun-3.0.3/configure.in	2013-02-16 23:44:44.004339680 +0100
@@ -166,7 +166,7 @@
    AC_MSG_RESULT()
    AC_CHECKING( for md5 Library and Header files ... )
 AC_SEARCH_HEADERS(md5.h,
-    $SSL_HDR_DIR /usr/include/openssl "" /usr/include /usr/include/ssl /usr/local/include /usr/local/ssl/include /usr/include/sys, 
+    $SSL_HDR_DIR "" , 
     ,
     AC_MSG_ERROR( SSL headers not found. )
    )
@@ -176,7 +176,7 @@
    AC_MSG_RESULT()
    AC_CHECKING( for blowfish Library and Header files ... )
    AC_SEARCH_HEADERS(blowfish.h,
-    $BLOWFISH_HDR_DIR /usr/include/ssl /usr/include/openssl /usr/include /usr/local/include /usr/local/ssl/include  /usr/include/crypto,
+    $BLOWFISH_HDR_DIR $SSL_HDR_DIR,
 	AC_CHECK_LIB(crypto, BF_set_key,
 	    [
 	       LIBS="$LIBS -lcrypto"
@@ -193,7 +193,7 @@
    AC_MSG_RESULT()
    AC_CHECKING( for AES Library and Header files ... )
    AC_SEARCH_HEADERS(aes.h,
-    $SSL_HDR_DIR /usr/include/ssl /usr/include/openssl /usr/include /usr/local/include /usr/local/ssl/include  /usr/include/crypto,
+    $SSL_HDR_DIR ,
 	AC_CHECK_LIB(crypto, AES_set_encrypt_key,
 	    [
 	       AC_DEFINE(HAVE_SSL_AES)
@@ -208,7 +208,7 @@
    AC_MSG_RESULT()
    AC_CHECKING( for EVP Library and Header files ... )
    AC_SEARCH_HEADERS(evp.h,
-    $SSL_HDR_DIR /usr/include/ssl /usr/include/openssl /usr/include /usr/local/include /usr/local/ssl/include  /usr/include/crypto,
+    $SSL_HDR_DIR ,
 	AC_CHECK_LIB(crypto, EVP_EncryptInit,
 	    [
 	       AC_DEFINE(HAVE_SSL_EVP)
